src/utils/commandParser.ts

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+import minimist = require('minimist');
+import _ = require('underscore');
+
+export function parse (args): ParsedCommand {
+	const argv = minimist(args);
+
+	const command: ParsedCommand = {
+		action: argv._[0],
+		attributes: argv._.slice(1),
+		flags: parseFlags().slice(1)
+	};
+
+	function parseFlags (): Array<Object> {
```

Comment:
This this `parseFlags` necessary?  Minimist is made for the purpose of parsing flags, it does everything you need already.  This seems like over-complication personally.